runner/client/src/mill/runner/client/MillProcessLauncher.java

```diff
@@ -114,12 +114,17 @@ static boolean isWin() {
   }
 
   static String javaHome() throws IOException {
-    String jvmId;
+    String jvmId = null;
     Path millJvmVersionFile = millJvmVersionFile();
 
     String javaHome = null;
     if (Files.exists(millJvmVersionFile)) {
       jvmId = Files.readString(millJvmVersionFile).trim();
+      if (jvmId.equals("global")) jvmId = null;
+    } else {
+      jvmId = "zulu:17.0.3";
```

Comment:
Is there a reason to use `zulu` rather than `temurin`? I recall defaulting to `zulu` sometimes, because it has JDK 8 builds for macOS ARM64 while `temurin` doesn't, but that shouldn't be a problem here.

Reply:
Here is the issue: Temurin doesn't publish JDKs for versions <21 on Windows-ARM:

- https://github.com/coursier/jvm-index/blob/master/indices/windows-arm64.json